src/selenium/cocPage.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-empty-label */
+'use strict';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['use strict' is unnecessary inside of modules. (strict)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)